src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ApplicationGatewayTests.cs

```diff
@@ -30,6 +30,13 @@ public ApplicationGatewayTests(ITestOutputHelper output)
 
         [Fact]
         [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestAvailableWafRuleSets()
+        {
+            NetworkResourcesController.NewInstance.RunPsTest(string.Format("Test-AvailableWafRuleSets -baseDir '{0}'", AppDomain.CurrentDomain.BaseDirectory));
```

Comment:
@jobatzil was there a test recording file associated with this test? I didn't see one. If so, make sure it is added in the test csproj file.